src/openforms/forms/tests/test_api_forms.py

```diff
@@ -51,6 +51,56 @@ def test_get_fields_is_not_filtered_for_staff(self):
             with self.subTest(field=field):
                 self.assertIn(field, FormSerializer.Meta.fields)
 
+    def test_auto_login_authentication_backend_validation_fails(self):
+        request = RequestFactory().get("/")
```

Comment:
please take the opportunity to fix this `RequestFactory` - it should be `rest_framwork.test.APIRequestFactory` since we're dealing with API calls and serializers.

I also vastly prefer having a proper end-to-end test instead of these `RequestFactory` specific tests, we care about the client `post`/`put`/`patch` behaviour, the implementation itself does not matter. So please convert to proper endpoint tests in `FormsAPITests`